agent/pkg/models/models.go

```diff
@@ -16,6 +16,14 @@ func GetEntry(r *tapApi.MizuEntry, v tapApi.DataUnmarshaler) error {
 	return v.UnmarshalData(r)
 }
 
+type EntriesRequest struct {
+	LeftOff   int    `form:"leftOff" validate:"required,min=0,max=2147483647"`
+	Direction int    `form:"direction" validate:"required,oneof='1' '-1'"`
+	Query     string `form:"query"`
+	Limit     int    `form:"limit" validate:"required,min=1,max=2147483647"`
+	TimeoutMs int    `form:"timeoutMs" validate:"required,min=1,max=2147483647"`
```

Comment:
I think the timeout can be optional too. in wasn't added we will put default value (something like couple of seconds)

Reply:
Done with https://github.com/up9inc/mizu/commit/c137437ee8326ad7acc71ee897459c4b794d5db0